.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
it's good practice to exclude your node_modules folder from the repo to cut down the size of everything. I know you created and pushed it super quick but just thought i'd point it out.